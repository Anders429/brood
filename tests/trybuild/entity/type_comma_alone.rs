use brood::Entity;

type Entity = Entity!(,);

fn main() {}
