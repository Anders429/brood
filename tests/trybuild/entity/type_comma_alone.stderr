error: no rules expected the token `,`
 --> tests/trybuild/entity/type_comma_alone.rs:3:23
  |
3 | type Entity = Entity!(,);
  |                       ^ no rules expected this token in macro call
